templates/theme/styles/base.less

```diff
@@ -308,6 +303,6 @@
   }
 }
 
-.editor.mini .scroll-view {
+:host(.mini) .scroll-view {
```

Comment:
In your code above was the change from `.mini` to `[mini]` intentional? I'll go ahead and assume so but figured it was worth double-checking.

Updated PR, shortly.